assurance/fixtures/patients/test-patient-2.json

```diff
@@ -0,0 +1,187 @@
+{
```

Comment:
I think this file can be removed, because the one we need is already  moved to the testing-harness directory.